istioctl/pkg/admin/istiodconfig.go

```diff
@@ -371,7 +371,7 @@ func (c *ControlzClient) GetScope(scope string) (*ScopeInfo, error) {
 var (
 	istiodLabelSelector = ""
 	istiodReset         = false
-	validationPattern   = `^[\w ]+:(debug|error|warn|info|debug)`
+	validationPattern   = `^[\w\- ]+:(none|error|warn|info|debug)`
```

Comment:
Can you please add the unit tests.
https://github.com/istio/istio/blob/master/istioctl/pkg/admin/istiodconfig_test.go